src/AbstractContainer.php

```diff
@@ -177,6 +177,11 @@ public function get()
         $path = func_get_args();
         $key  = implode('.', $path);
 
+        // allow dot-notation in first parameter
+        if (count($path) === 1 && strstr($path[0], '.')) {
+            $path = explode('.', $path[0]);
+        }
```

Comment:
See https://github.com/rtheunissen/config/issues/1.

`implode('.', $path);` takes care of it already, doesn't it?